src/mmw/js/src/core/opacityControl.js

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
> This should probably go in https://github.com/WikiWatershed/model-my-watershed/tree/develop/src/mmw/js/shim

At one time I had it there, but I decided to move it to its current place because I noticed that it does something very analogous to `layerControl.js`.